test/test_transforms.py

```diff
@@ -298,6 +300,9 @@ def test_pil_to_tensor_errors(self):
         trans = transforms.PILToTensor()
         np_rng = np.random.RandomState(0)
 
+        # TODO: DID NOT RAISE
+        return
```

Comment:
For the `np.array(...)` this is the same as https://github.com/pytorch/vision/pull/7159/files#r1108438047. For plain `np.array(...)`'s, `PILToTensor` will just pass them through

https://github.com/pytorch/vision/blob/3991ab994630ede87706dc9b46d005b8834cc5bc/torchvision/transforms/v2/_type_conversion.py#L13-L14

whereas `ToTensor` can handle them

https://github.com/pytorch/vision/blob/3991ab994630ede87706dc9b46d005b8834cc5bc/torchvision/transforms/v2/_deprecated.py#L12-L13

Thus, both individual calls aren't failing here. I'm ok with that behavior.